posh-git.psm1

```diff
@@ -92,6 +92,57 @@ if (!$currentPromptDef -or ($currentPromptDef -eq $defaultPromptDef)) {
     Set-Item Function:\prompt -Value $poshGitPromptScriptBlock
 }
 
+# If running interactive, check if user want's `import-module posh-git` added to their profile
+if (!$NoProfileCheck -and ($MyInvocation.ScriptName.Length -eq 0)) {

```

Comment:
Is the `Length` check more idiomatic than `!$MyInvocation.ScriptName`?

Also, s/want's/wants above. :trollface: